.github/workflows/delete_review_app_aks.yml

```diff
@@ -0,0 +1,43 @@
+name: Delete review app on AKS
+
+on:
+  pull_request:
+    branches:
+    - main
+    types:
+    - closed
+
+concurrency: deploy-${{ github.ref }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: review
+
```

Comment:
we normally only run if there was a deploy label on the PR, and also check for a state file which is also deleted as part of this workflow. see https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/.github/workflows/delete-review-app.yml

Reply:
Added condtion

Reply:
Fixed review